### PR TITLE
DO-340: No occasion message fixed.

### PR DESCRIPTION
**Pull request by amitjain24524**

### CODE REVIEW

File: src/components/member/MembersOccasion/MembersOccasion.jsx [hunk 1]
```diff
@@ -122,7 +121,7 @@ class MembersOccasion extends React.Component {
           onClick={this.onClickOccasionCount}
         />
         <Container className="member-occasions-container">
-          <NoOccasion members={membersToDisplay} occasionType={occasionType} />
+          <NoOccasion members={membersToDisplay} occasionType={occasionType} countType={countType} />
```
Comment (mohitsaladiya) [inline src/components/member/MembersOccasion/MembersOccasion.jsx:124]:
```suggestion
          <NoOccasion members={membersToDisplay} occasionType={occasionType} timePeriod={countType} />
```

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx [hunk 2]
```diff
@@ -14,21 +14,37 @@ const OccasionsType = (props) => {
...
       <Container className="no-occasions__message">
-        {props.message}
+        {renderMessageText(props)}
       </Container>
     </Container>
   );
 };
 
+const renderMessageText = (props) => {
+  if (props.countType === TIME_PERIOD.TODAY) {
```
Comment (mohitsaladiya) [inline src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx:24]: Rename the countType prop to timePeriod

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx [hunk 3]
```diff
@@ -14,21 +14,37 @@ const OccasionsType = (props) => {
...
+  }
+};
+
 const renderNoOccasionContent = (props) => {
-  const { members = [], occasionType } = props;
+  const { members = [], occasionType, countType } = props;
     if (occasionType === OCCASION_TYPE.BIRTHDAY) {
       return (
-        <OccasionsType icon={<IconCake />} message={NO_OCCASION_TYPE_MESSAGE.BIRTHDAY_MESSAGE} iconClass="birthday-icon" />
+        <OccasionsType icon={<IconCake />} message={OCCASION_TYPE.BIRTHDAYS} countType={countType} iconClass="birthday-icon" />
```
Comment (mohitsaladiya) [inline src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx:43]: Why are we passing OCCASION_TYPE.ANNIVERSARIES in message??
```suggestion
        <OccasionsType icon={<IconCake />} timePeriod={timePeriod} iconClass="birthday-icon" />
```

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx [hunk 4]
```diff
@@ -14,21 +14,37 @@ const OccasionsType = (props) => {
...
+      <>
+        {NO_OCCASION_TYPE_MESSAGE.MESSAGE_FIRST} {props.message} {NO_OCCASION_TYPE_MESSAGE.MESSAGE_SECOND} {props.countType}{NO_OCCASION_TYPE_MESSAGE.MESSAGE_END}
+      </>
+    );
+  }
+};
+
 const renderNoOccasionContent = (props) => {
-  const { members = [], occasionType } = props;
+  const { members = [], occasionType, countType } = props;
```
Comment (mohitsaladiya) [inline src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx:40]:
```suggestion
  const { members = [], occasionType, timePeriod} = props;
```

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx [hunk 5]
```diff
@@ -14,21 +14,37 @@ const OccasionsType = (props) => {
...
     if (occasionType === OCCASION_TYPE.BIRTHDAY) {
       return (
-        <OccasionsType icon={<IconCake />} message={NO_OCCASION_TYPE_MESSAGE.BIRTHDAY_MESSAGE} iconClass="birthday-icon" />
+        <OccasionsType icon={<IconCake />} message={OCCASION_TYPE.BIRTHDAYS} countType={countType} iconClass="birthday-icon" />
       );
     } else if (occasionType === OCCASION_TYPE.ANNIVERSARY) {
       return (
-        <OccasionsType icon={<IconRing />} message={NO_OCCASION_TYPE_MESSAGE.ANNIVERSARY_MESSAGE} iconClass="anniversary-icon" />
+        <OccasionsType icon={<IconRing />} message={OCCASION_TYPE.ANNIVERSARIES} countType={countType} iconClass="anniversary-icon" />
```
Comment (mohitsaladiya) [inline src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx:47]: **Why are we passing OCCASION_TYPE.ANNIVERSARIES in message??**
```suggestion
        <OccasionsType icon={<IconRing />} timePeriod={timePeriod} iconClass="anniversary-icon" />
```

File: src/constants/common/commonConstants.js [hunk 6]
```diff
@@ -216,13 +216,16 @@ export const DASHBOARD_CARD_ICON = {
...
   ANNIVERSARY: 'Anniversary',
+  BIRTHDAYS: 'Birthdays',
+  ANNIVERSARIES: 'Anniversaries',
 };
 
 export const PWA_TEXT = {
   PWA_ICON_NAME: 'Install App',
 };
 
 export const NO_OCCASION_TYPE_MESSAGE = {
```
Comment (mohitsaladiya) [inline src/constants/common/commonConstants.js:227]: Move it to json.

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx [hunk 7]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import './_NoOccasions.scss';
-import { NO_OCCASION_TYPE_MESSAGE, OCCASION_TYPE } from '../../../../constants/common/commonConstants';
+import { NO_OCCASION_TYPE_MESSAGE, OCCASION_TYPE, TIME_PERIOD } from '../../../../constants/common/commonConstants';
```
Comment (mohitsaladiya) [inline src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx:5]: Get the NO_OCCASION_TYPE_MESSAGE from json instead

File: src/components/member/MembersOccasion/MembersOccasion.jsx (hunk 1 above)
Comment (amitjain24524) [reply src/components/member/MembersOccasion/MembersOccasion.jsx]: done

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx (hunk 2 above)
Comment (amitjain24524) [reply src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx]: done

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx (hunk 3 above)
Comment (amitjain24524) [reply src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx]: For finding the type of occasion birthday or anniversary

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx (hunk 4 above)
Comment (amitjain24524) [reply src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx]: done

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx (hunk 5 above)
Comment (amitjain24524) [reply src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx]: For finding the type of occasion birthday or anniversary

File: src/constants/common/commonConstants.js (hunk 6 above)
Comment (amitjain24524) [reply src/constants/common/commonConstants.js]: done

File: src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx (hunk 7 above)
Comment (amitjain24524) [reply src/components/member/MembersOccasion/NoOccasions/NoOccasions.jsx]: done

**Pull request merged**